src/Autosuggest.js

```diff
@@ -156,6 +157,18 @@ class Autosuggest extends Component {
     }
   }
 
+  maybeSelectFirstSuggestion(event, value) {
+    const { selectFirstSuggestion, multiSection, updateFocusedSuggestion } = this.props;
+
+    if (selectFirstSuggestion) {
+      let sectionIndex = multiSection ? 0 : null;
+      let suggestionIndex = 0;
+
+      updateFocusedSuggestion(sectionIndex, suggestionIndex);
+      this.maybeCallOnChange(event, value, 'auto');
```

Comment:
To mirror the call to `maybeCallOnChange` in the up/town events. I.e. the system is auto-selecting the first item, which is a change event, and clients will want to handle the selection change appropriately. Let me know if that makes sense.